pydss/flopper.py

```diff
@@ -126,10 +126,10 @@ def dent(self, bid_id, lot, bid):
     self.bids[bid_id].tic = int(self.now)+ self.ttl
 
 
-def deal(self, sid):
+def deal(self, sid,lot):
     require(self.live == 1,"Flapper/not-live")
     require(self.bids[sid].tic != 0 and (self.bids[sid].tic < self.now or self.bids[sid].end < self.now), "Flapper/not-finished")
```

Comment:
looks like you use `self.now` here, be sure to instead accept `now` as a parameter to the function and use it accordingly within